### PR TITLE
pd-recovery: add the new way to recover

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Users can recover the PD from a PD node.

Translate from https://github.com/pingcap/docs-cn/pull/12186

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12186
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-recover.md [hunk 1]
```diff
@@ -21,13 +21,53 @@ PD Recover is a disaster recovery tool of PD, used to recover the PD cluster whi
...
+```shell
+./bin/pd-server --force-new-cluster --name=pd-127.0.0.10-2379 --client-urls=http://0.0.0.0:2379 --advertise-client-urls=http://127.0.0.1:2379 --peer-urls=http://0.0.0.0:2380 --advertise-peer-urls=http://127.0.0.1:2380 --config=conf/pd.toml
+```
+
+### Step 3: Repair metadata using `pd-recover`
+
+Since this method relies on a minority PD node to recover the service, the node might contain outdated data. If the `alloc_id` and `tso` data roll back, the cluster data might be corrupted or unavailable. To prevent this, you need to use `pd-recover` to modify the metadata to ensure that the node can provide correct allocation IDs and TSO services. The following is an example:
+
+```shell
+./bin/pd-recover --from-old-member --endpoints=http://127.0.0.1:2379 // Specify the corresponding PD address
```
Comment (ran-huang) [inline pd-recover.md:47]:
```suggestion
./bin/pd-recover --from-old-member --endpoints=http://127.0.0.1:2379 # Specify the corresponding PD address
```

File: pd-recover.md [hunk 2]
```diff
@@ -21,13 +21,53 @@ PD Recover is a disaster recovery tool of PD, used to recover the PD cluster whi
...
+>
+> Additionally, `pd-recover` does not modify the TSO. Therefore, before performing this step, make sure that the local time is later than the time when the failure occurs, and verify that the NTP clock synchronization service is enabled between the PD components before the failure. If it is not enabled, you need to adjust the local clock to a future time to prevent the TSO from rolling back.
+
+### Step 4: Restart the PD node
+
+Once you see the prompt message `recovery is successful`, restart the PD node.
+
+### Step 5: Scale out PD and start the cluster
+
+Scale out the PD cluster using the deployment tool and start the other components in the cluster. At this point, the service is available.
```
Comment (ran-huang) [inline pd-recover.md:62]:
```suggestion
Scale out the PD cluster using the deployment tool and start the other components in the cluster. At this point, the PD service is available.
```

File: pd-recover.md (hunk 1 above)
Comment (ran-huang) [inline pd-recover.md:47]:
```suggestion
./bin/pd-recover --from-old-member --endpoints=http://127.0.0.1:2379 # Specify the corresponding PD address
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Comment by Oreoxmt** [comment]
@nolouch PTAL

**Pull request merged**